main.py

```diff
@@ -1,12 +1,210 @@
+    #     print("You must select a customer before updating it or deleting it.")
+    #     print("Let's select a customer")
+    #     choice = "10"
+
+    return choice
+
+def print_stars():
+    print("*******************************************************\n")
+
+def run_cli_video_store():
+    video_store = Videostore(url=VIDEO_STORE_URL)
+
+    options = list_options()
+
+    play = True
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, video_store)
+
+        # video_store.print_selected_video()
+        # video_store.print_selected_customer()
+
+
+# ==========All For Video Options========== 
+
+        # Option 1: create a video
+        if choice == '1':
+            print("Great! Let's create a new video.")
+            title = input("Title of the video: ")
+            release_date = input("The release date of the video: ")
+            total_inventory = input("The total Inventory: ")
+            response = video_store.create_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print(f"New video: {title} has been added.") 
+        
+        # Option 2: update a video
+        elif choice == '2':
+            print("Great! Let's update the video")
+            title = input("New title of the video: ")
+            release_date = input("New release date of the video: ")
+            total_inventory =input("New total inventory: ")
+            response = video_store.update_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print(f"Updated video: {title} has been updated.") 
+        
+        # Option 3: Delete a video 
+        elif choice=='3':
+            delete_video_id = input("Enter video id to delete: ")
+            if delete_video_id.isnumeric():
+                video_store.delete_video(delete_video_id)
+                print(f"Video id {delete_video_id} has been deleted.")
+                
+
+        # Option 4: Get information of all videos
+        elif choice=='4':
+            print("Videos: ")
+            print_stars()
+            for video in video_store.list_videos():
+                print(video)
+
+        # Option 5: Get information of one video 
+        elif choice=='5':
+            select_by = input("Select by video ID: ")
+            if select_by.isnumeric():
+                video = video_store.get_video(id=video_id)
+                print(video)
+
+
+# ==========All For Customers Options========== 
+
+        # Option 6: Create a new customer 
+        elif choice == '6':
+            print("Great! Let's create a new customer.")
+            name = input("Name of the customer: ")
+            postal_code = input("Customer's postal_code: ")
+            phone = input("Customer's phone number: ")
+
+            response = video_store.create_customer(name=name, postal_code=postal_code, phone=phone) 
+            print("New customer: #{response['id']} has been added.") 
```

Comment:
This will result in an error getting raised when the api responds with an error code because the response will not have an 'id' key in that case. See the comment on the create_customer method for more context.